performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationResponse.java

```diff
@@ -3,32 +3,66 @@
 import android.os.Bundle;
 import android.os.Parcel;
 import android.os.Parcelable;
-import com.google.gson.annotations.SerializedName;
+import android.support.annotation.NonNull;
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.Map;
+import org.json.JSONException;
+import org.json.JSONObject;
 
-/** Configuration Result */
-class ConfigurationResult implements Parcelable {
+/**
+ * Configuration Result
+ */
+class ConfigurationResponse implements Parcelable {
```

Comment:
Why does this still need to implement `Parcelable`? I thought it was only needed because of GSON?

- [ ] Remove Parcelable implementation if possible

Reply:
yes you are right, we never needed the parceling to begin with, i don't think GSON has any relation to `Parcelable`.